landoapi/landing_worker.py

```diff
@@ -268,154 +287,126 @@ def run_job(self, job, repo, hgrepo, treestatus, patch_bucket):
                     )
                     self.notify_user_of_landing_failure(job)
                     return True
+                patch_bufs.append((revision_id, patch_buf))
 
-                # Download all patches locally from S3.
```

Comment:
Why is this patch so girthy if most of it is an indentation change? Weird :thinking:

Reply:
It's mostly undoing a previous patch that was also an indentation change. Just the way that GitHub parses the diff I guess?